openweathermap/weather.json

```diff
@@ -11,6 +11,9 @@
             "q": true
         }
     },
+    "examples": {
+        "q": "'vancouver'"
```

Comment:
why there are single quotes in all the examples?

Reply:
Good question. This is so the info function displays single quotes around the example value for the parameter. If they aren't included then vancouver wouldn't be displayed with single quotes. Let me know if this makes sense.